packages/gatekeeper/src/components/callback/tests/implicit.test.tsx

```diff
@@ -148,11 +148,10 @@ describe('gatekeeper/implicit/OauthCallback', () => {
 
     const { onadataAuth } = helperFixtures;
     const url = 'https://stage-api.ona.io/api/v1/user.json';
-    const hash =
-      '#access_token=iLoveOov&expires_in=36000&token_type=Bearer&scope=read+write&state=abc';
+    const href = 'http://localhost/';
```

Comment:
Let's include the hash `#access_token=iLoveOov&expires_in=36000&token_type=Bearer&scope=read+write&state=abc` in this url